src/serena/agent.py

```diff
@@ -544,7 +565,7 @@ def apply(self, relative_path: str, recursive: bool, max_answer_chars: int = _DE
             os.path.join(self.project_root, relative_path),
             relative_to=self.project_root,
             recursive=recursive,
-            ignored_dirs=self.project_config.ignored_dirs,
+            ignored_dirs=self.project_config.ignored_paths,
```

Comment:
Shouldn't this be updated to use the full set of ignored paths?

Reply:
Yes, let me do that in a followup PR, ok?